app/templates/index.hbs

```diff
@@ -38,20 +38,19 @@
             <th colspan="3">Acties</th>
```

Comment:
not important, but I changed this yesterday. This can be set back to "2" as the edit button moved :)

Reply:
No good catch thanks!